docs/pages/setup/reference/backends.mdx

```diff
@@ -216,15 +216,15 @@ running on an EC2 instance with an IAM role.
 You can add optional query parameters to the S3 URL. The Teleport Auth
 Service reads these parameters to configure its interactions with S3:
 
-`s3://bucket/path?region=us-east-1&endpoint=mys3.example.com&insecure=false&disablesse=false&acl=private`
+`s3://bucket/path?region=us-east-1&endpoint=mys3.example.com&insecure=false&disablesse=false&acl=private&use_fips_endpoint=true`
 
 - `region=us-east-1` - set the Amazon region to use.
 - `endpoint=mys3.example.com` - connect to a custom S3 endpoint.
 - `insecure=true` - set to `true` or `false`. If `true`, TLS will be disabled.
 - `disablesse=true` - set to `true` or `false`. If `true`, S3 server-side encryption will be disabled. If `false`, aws:kms (Key Management Service) will be used for server-side encryption. Other SSE types are not supported at this time.
 - `sse_kms_key=kms_key_id` - If set to a valid AWS KMS CMK key ID all objects uploaded to S3 will be encrypted with this key. Details can be found below.
 - `acl=private` - set the [canned ACL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html#canned-acl) to use. Must be one of the predefined ACL values.
-
+- `use_fips_endpoint=true` -  set to `true` or `false`. If `true`, [FIPS](https://aws.amazon.com/compliance/fips/) S3 endpoints will be used. If `false`, normal S3 endpoints will be used. If unset, the AWS Environment Variable `AWS_USE_FIPS_ENDPOINT` will determine which endpoint is used. FIPS endpoints will also be used if Teleport is run with the `--fips` flag.
```

Comment:
There seem to be three possible values for the `AWS_USE_FIPS_ENDPOINT` variable: https://github.com/aws/aws-sdk-go/blob/258263306d5faf0fc5f629dfcccf0f5ffedca131/aws/endpoints/endpoints.go#L35-L44

Should we talk about the difference between unsetting and disabling FIPS endpoint resolution in the context of the environment variable?

Reply:
I certainly think that's a valid conversation to have. It also may be helpful to specify the priority order for the different options, ie; `QueryParam > --fips flag > ENVVAR`. Do you want me to take a spin at trying to describe all of this succinctly? I was intentionally trying to avoid being too verbose with this.